docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -221,6 +221,12 @@ To update the logos, put image files in the `./src/admin/extensions` folder and
 
 The size of the custom image should be the same as the default one (434px x 120px).
```

Comment:
Is there still a size constraint? It is still mentioned here in the dev docs, but it is not in the User Guide documentation.

Reply:
Ah, good question. Maybe we should ask the front-end devs? 🤔

Reply:
Ah yes good point! There's no size limit when setting a logo via the config files but there's one when uploading a logo via the admin panel. Let me update the docs.